### DISCUSSION TITLE
Help with understanding rendering forms.

### DISCUSSION THREAD

**Discussion by rwxzig** [OrchardCMS/OrchardCore#14937] [Q&A] [answered]
This is a very important project for me and I want to get it right. I have been looking at orchard but had to take a break because of my exams at uni. Now I am back and I want to continue learning Orchard. I have been struggling with understanding the rendering model. The samples are helping tremendously, but I still have some missing parts.

I created these types and its corresponding files.
ManyForms -> Content-ManyForms.liquid
FormTx -> Content-FormTx.liquid
ManyForms-ListPart.liquid // To render the list of the items in the ManyForms page

The hierarchy of the types is as follows:
The page called ManyForms has:
![image](https://github.com/OrchardCMS/OrchardCore/assets/42944941/8b9182e2-7603-4c3c-8e70-3ff6b254f5b2)
The list is of type FormTx

And here is the FormTx definition, similarly to BlogPost, it has Tags, Subtitle and Category. And also:
![image](https://github.com/OrchardCMS/OrchardCore/assets/42944941/606ba671-4fef-490e-89d5-b75d758bf051)

Essentially a FormTx has a title, subtitle to indicate what this form is form, and it can have tags and category.
But I have hit a wall on trying to render the Form. 

I think this is not the best solution, and I don't have a clear idea on how to implement this, but it is a start and I want to see where it will lead me, but you are more than welcome to suggest any changes or different way on how to implement this. 

Just to see how it looks on Content-FormTx.liquid
![image](https://github.com/OrchardCMS/OrchardCore/assets/42944941/18a62e36-87d4-4840-a45b-65d9b2116a2f)

And this is how it looks in the console
![image](https://github.com/OrchardCMS/OrchardCore/assets/42944941/d8cc7f5e-1eb7-44e7-a61c-381b80770591)

This is my repo, if you can make some very quick changes, I would greatly appreciated.
https://github.com/cptzig/OrchardCore.FormsMD

As I said this is very important project for me, so I can even pay some money (reasonably) just to learn this properly. 

EDIT: I am using pages to make new forms, as it was demo-ed from Lombiq in the OrchardCore tutorial. 
And now I am looking into modules to overcome the limitations and extend functionalities.

**Comment by ns8482e** [comment, marked as answer]
You have content item FormTx with FormPart but you have nothing in it to render. Attach a FlowPart so you could  add form widgets like label, input, button to the form.

Also Please share the rendered html
